ops/model.py

```diff
@@ -1884,7 +1884,7 @@ def add_metrics(self, metrics, labels=None):
         cmd.extend(metric_args)
         self._run(*cmd)
 
-    def get_pebble(self, socket_path: str) -> 'pebble.Client':
+    def get_pebble(self, container: str, socket_path: str) -> 'pebble.Client':
```

Comment:
Maybe I'm being dense, but this argument doesn't seem to actually get used anywhere?

Reply:
Ah, looks like this is in place to keep the interface the same across real pebble and harness pebble?

Reply:
Yep

Reply:
Yeh, I just realised I failed to come back on this. This *does* feel a little uncomfortable to me - and it'll certainly appear odd to others reading the codebase that there is a random unused argument here (as it did to me until I read the whole PR..)

Reply:
I've reverted this (private) api change.